src/wallet/rpc/addresses.cpp

```diff
@@ -534,7 +534,7 @@ RPCHelpMan getaddressinfo()
                         {RPCResult::Type::BOOL, "solvable", "If we know how to spend coins sent to this address, ignoring the possible lack of private keys."},
                         {RPCResult::Type::STR, "desc", /*optional=*/true, "A descriptor for spending coins sent to this address (only when solvable)."},
                         {RPCResult::Type::STR, "parent_desc", /*optional=*/true, "The descriptor used to derive this address if this is a descriptor wallet"},
-                        {RPCResult::Type::BOOL, "isscript", "If the key is a script."},
+                        {RPCResult::Type::BOOL, "isscript", /*optional=*/true, "If the key is a script."},
```

Comment:
unrelated: that is _quite_ the field description 😳 Time to start thinking about deprecating `isscript` and `iswitness`, perhaps?

Reply:
yeah I was having a real hard time understanding what a lot of these fields actually mean...